app/controllers/dor_controller.rb

```diff
@@ -52,11 +40,13 @@ def reindex_pid(pid, logger:, add_attributes:)
         Failure(:conversion_error)
       end
     end.format('%n realtime %rs total CPU %ts').gsub(/[()]/, '')
-
+    logger.info "document found, now generating document solr"
```

Comment:
Here and L48: Should these be `Rails.logger`?

Reply:
the `logger` method is already on all controllers.